css/amp.css

```diff
@@ -78,6 +78,15 @@ body {
   right: 0;
 }
 
+.-amp-layout-flex-item {
+  display: block;
+  position: relative;
+  flex: 1 1 0;
```

Comment:
Ok, let me rephrase this. We need to confirm these properties:

1. `flex` already includes `flex-basis`. In our case it's 0. Should `flex` be actually be `flex: 1 1 auto`?
2. Why do we need `align-self: stretch`?
3. Do we even need width and height `100%` here? What happens without them? My initial idea behind them is to break layout when parent is NOT `display: flex`. I think it works and doesn't interfere when parent is as expected `display:flex`. But we need to make sure it works as expected.